src/utils/btc.ts

```diff
@@ -0,0 +1,12 @@
+import type { TChainWallet } from '..'
+import { btcChainWalletMap } from '..'
+import { getConnectedWallets } from './common'
+import type { XDeFi } from '@/provider'
+
+const getBTCConnectedWallets = async (xDeFiProvider: XDeFi) => {
+  return await getConnectedWallets(btcChainWalletMap, async ({ name }: TChainWallet) => {
+    return await xDeFiProvider.getProviderByKey(name as any).getAccounts()
```

Comment:
as far as I know, you don't need to use `await` in `return`. Webstorm usually warns about this

Reply:
ty!